CHANGELOG.md

```diff
@@ -9,6 +9,21 @@ The format is based on [Keep a Changelog].
   [#465]).
 * The experimental `selectrum-helm.el` package has been removed
   ([#313], [#466]).
+* The faces `selectrum-primary-highlight` and
+  `selectrum-secondary-highlight` have been deprecated and
+  subsequently removed. The match highlighting should be configured
```

Comment:
We haven't made a release with the deprecated faces so can we say they  have been deprecated?

Reply:
Yes, this is one of the cases where we should have made a 3.2 first, deprecating the faces and then remove them in 4.0. But I think faces are non-problematic. There won't be serious bugs due to missing faces. If you use undefined face symbols nothing happens. The only case where something breaks is if someone uses `(set-face-attribute 'foo nil :inherit 'default)` on an unknown face `foo`.

Reply:
One thing I should add - Since we release a 4.0 next, we can actually do arbitrary breaking changes if you use whatever semantic versioning. It is not nice to have no intermediate transitional package for certain changes, but it is also not a blocker. I would not worry about this particular case with faces at least.

Reply:
I agree, we don't need to worry to much about the faces, but maybe we can simplify the sentence then that they got removed.

Reply:
Sure, feel free to change this in any way you like!

Reply:
Okay, thanks again for updating!